### ISSUE TITLE
PARTIAL syntax in DQL is deprecated

### ISSUE THREAD

**Issue by simonschaufi** [kimai/kimai#4383]
### Describe the issue

PHP Deprecated: PARTIAL syntax in DQL is deprecated. (Parser.php:1850 called by Parser.php:2281, https://github.com/doctrine/orm/issues/8471, package doctrine/orm) in /kimai2/vendor/doctrine/deprecations/lib/Doctrine/Deprecations/Deprecation.php on line 210

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.1.0

### How do you run Kimai?

Shared-Hosting

### Which PHP version are you using?

8.1

Maybe this would help? https://github.com/doctrine/orm/pull/10920

**Comment by kevinpapst** [comment]
Yep. There is no replacement yet, the Doctrine team deprecated an essential feature without providing a way to rebuild it differently yet. Unfortunately there is no way that I know of, that can fix this in a meaningful way and without a performance hit...

This might be the solution in the future: https://github.com/doctrine/orm/pull/8391

**Comment by schka17** [comment]
> ### Describe the issue
> PHP Deprecated: PARTIAL syntax in DQL is deprecated. (Parser.php:1850 called by Parser.php:2281, [doctrine/orm#8471](https://github.com/doctrine/orm/issues/8471), package doctrine/orm) in /kimai2/vendor/doctrine/deprecations/lib/Doctrine/Deprecations/Deprecation.php on line 210
> 
> ### I already tried
> * [x]  I've read and searched [the documentation](https://www.kimai.org/documentation/).
> * [x]  I've searched for similar issues in this repository.
> * [x]  I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).
> 
> ### Kimai version
> 2.1.0
> 
> ### How do you run Kimai?
> Shared-Hosting
> 
> ### Which PHP version are you using?
> 8.1
> 
> ### Logfile
> _No response_
> 
> ### Screenshots
> _No response_
> 
> Maybe this would help? [doctrine/orm#10920](https://github.com/doctrine/orm/pull/10920)

Hi there,
just ran into the same error.

Found out the root cause was the wrong configuration from the SAML provider, in my case Entera. I was missing claims. 
Made all the configuration according to the (outdated) documentation, added groups and employeeid and now it works.

![image](https://github.com/kimai/kimai/assets/11760789/ca093afd-19f8-4b5d-8af1-57582bd1a832)

**Comment by kevinpapst** [comment]
I did my best in the past to get rid of PARTIAL queries, but they are necessary at some places.

Luckily this feature was un-deprecated by the Doctrine Team.
